user-contract-mon/internal/monitor.go

```diff
@@ -0,0 +1,66 @@
+package monitor
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/NicoNex/echotron/v3"
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/deployer"
+)
+
+type Monitor struct {
+	Bot      echotron.API
+	Mnemonic string `env:"MNEMONIC"`
+	BotToken string `env:"BOT_TOKEN"`
+	Network  string `env:"NETWORK"`
+	interval int    `env:"INTERVAL"`
+}
+
+func NewMonitor(envPath string) (Monitor, error) {
+	mon := Monitor{}
+	envMap, err := parseConfig(envPath)
+	if err != nil {
+		return mon, err
+	}
+	for key, value := range envMap {
```

Comment:
I don't think you need to loop over the map, you can just access it like `envMap[key]`